python/kserve/requirements.txt

```diff
@@ -16,7 +16,7 @@ cloudevents>=1.2.0
 avro>=1.10.1
 boto3==1.20.24
 psutil>=5.0
-ray[serve]==1.9.0
+ray[serve]==1.9.2
```

Comment:
this is also the version in latest kserve master ([link](https://github.com/kserve/kserve/blob/7e74dca4421af26cfbe96dccdf3806e77a63c1b3/python/kserve/requirements.txt#L20))

Reply:
if it's already in the master branch, shouldn't we just build the image from it? not sure why do we need duplicating the changes

Reply:
or cherry-pick the existing commit to [kaptain/v0.8.0](https://github.com/mesosphere/kfserving/tree/kaptain/v0.8.0) branch

Reply:
yeah that's what I want to do.
the latest kserve images report 1 extra critical CVE. I think it's worth merging this to `kaptain/v0.8.0` and building from there